src/main/java/uk/gov/hmcts/divorce/caseworker/service/print/AosPackPrinter.java

```diff
@@ -61,9 +62,49 @@ public void sendAosLetterToRespondent(final CaseData caseData, final Long caseId
         }
     }
 
+    public void sendJudicialSeparationAoSLetterToRespondent(final CaseData caseData, final Long caseId) {
+
+        final List<Letter> currentAosLetters = aosLetters(caseData, NOTICE_OF_PROCEEDINGS_APP_2);
+
+        if (!isEmpty(currentAosLetters)) {
+
+            final String caseIdString = caseId.toString();
+            final Print print = new Print(currentAosLetters, caseIdString, caseIdString, LETTER_TYPE_RESPONDENT_PACK);
+
+
+            final UUID letterId = bulkPrintService.printAosRespondentPack(print, false);
+            log.info("Letter service responded with letter Id {} for case {}", letterId, caseId);
+        } else {
+            log.warn(
+                "AoS Pack print for respondent has missing documents. Expected documents with type {} , for Case ID: {}",
+                List.of(APPLICATION, NOTICE_OF_PROCEEDINGS_APP_2),
+                caseId);
+        }
+    }
+
+    public void sendJudicialSeparationAoSLetterToApplicant(final CaseData caseData, final Long caseId) {
+
+        final List<Letter> currentAosLetters = aosLetters(caseData, NOTICE_OF_PROCEEDINGS_APP_1);
+
+        if (!isEmpty(currentAosLetters)) {
+
+            final String caseIdString = caseId.toString();
+            final Print print = new Print(currentAosLetters, caseIdString, caseIdString, LETTER_TYPE_RESPONDENT_PACK);
+
+
+            final UUID letterId = bulkPrintService.printAosRespondentPack(print, false);
```

Comment:
We are calling print respondent pack in this method, should we be using an applicant method instead (as this is for applicant1 not applicant2). Also the letterpack ID may need to be updated to reflect JS. 

Personal note: Again we need to add exception in else statement. See previous comment.